File: app/src/main/java/dev/ragnarok/fenrir/db/column/TempDataColumns.java
package dev.ragnarok.fenrir.db.column;

import android.provider.BaseColumns;


public class TempDataColumns implements BaseColumns {

    public static final String TABLENAME = "temp_app_data";

    public static final String OWNER_ID = "owner_id";
    public static final String SOURCE_ID = "source_id";
    public static final String DATA = "data";

    // This class cannot be instantiated
    private TempDataColumns() {
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/db/column/TempDataColumns.kt
package dev.ragnarok.fenrir.db.column

import android.provider.BaseColumns

object TempDataColumns : BaseColumns {
    const val TABLENAME = "temp_app_data"
    const val OWNER_ID = "owner_id"
    const val SOURCE_ID = "source_id"
    const val DATA = "data"
}